.github/workflows/build-and-test.yml

```diff
@@ -71,25 +71,24 @@ jobs:
   setup_test:
     name: Run setup test
     runs-on: ubuntu-latest
-    container: precice/ci-ubuntu-1804
     steps:
       - name: Checkout Repository
         uses: actions/checkout@v2
-      - name: Install & upgrade pip3
+      - name: Install & upgrade pip3 and install OpenMPI
         run: |
-          apt-get -yy update
-          apt-get install -y python3-pip
-          rm -rf /var/lib/apt/lists/*
+          sudo apt-get -yy update
+          sudo apt-get install -y python3-pip libopenmpi-dev
+          sudo rm -rf /var/lib/apt/lists/*
```

Comment:
Related to #101: We should be able to remove the mpi part here.

But for the current state everything is fine.

Reply:
Okay, will keep this dependency problem in mind.